workspace/workspace.go

```diff
@@ -251,26 +257,27 @@ func LoadWorkspace() (Workspace, error) {
 
 // validWorkspaceConfig will check if every expected part of whole Configuration Graph is initialized
 // as expected
-func validWorkspaceConfig(workspace Workspace) (valid bool) {
+func validWorkspaceConfig(workspace Workspace) error {
+	var valid bool
 	if valid = len(workspace.packages) > 0; !valid {
-		return
+		return fmt.Errorf("No packages in this workspace")
```

Comment:
nit: Go error messages should generally start with lowercase (here and elsewhere) https://github.com/golang/go/wiki/CodeReviewComments#error-strings